cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -118,6 +119,26 @@ nullTracers = Tracers {
       muxTracer = nullTracer
     }
 
+
+instance ElidingTracer
+  (WithSeverity (WithTip blk (ChainDB.TraceEvent blk))) where
+  -- equivalent by type and severity
+  isEquivalent (WithSeverity s1 (WithTip _tip1 (ChainDB.TraceLedgerReplayEvent _ev1)))
```

Comment:
If they are both `TraceLedgerReplayEvent` and have the same severity? No need to match on the actual event of `TraceLedgerReplayEvent`?

Reply:
equivalence just by comparing severity and the type; could be made more complicated..